PayrailsCSE/Sources/PayrailsCSE/PayrailsCSE.swift

```diff
@@ -0,0 +1,273 @@
+import Foundation
+import JOSESwift
+
+struct InitResponse: Codable {
+    let version: String
+    let data: String
+}
+
+struct Card: Codable {
+    let holderName: String?
```

Comment:
Holder reference is also mandatory now here:
```suggestion
    let holderReference: String
    let holderName: String?
```